hvac/api/secrets_engines/kv.py

```diff
@@ -70,6 +70,8 @@ def __getattr__(self, item):
         :return: The selected secrets_engines class corresponding to this instance's default_kv_version setting
         :rtype: hvac.api.vault_api_base.VaultApiBase
         """
+        if item in "_default_kv_version":
```

Comment:
should this be an equality check instead? or is it meant to be checking for multiple possible strings like both `default_kv_version` and `_default_kv_version`?

If the latter, I think I'd prefer a comment at a minimum, but probably something more explicit instead, like using `in ("default_kv_version", "_default_kv_version")` or two separate `if` statements, or maybe even `item.lstrip("_") == "default_kv_version"`?

Reply:
It was indeed to check for both `default_kv_version` and `_default_kv_version`, I just updated this bit so that it is more explicit.